internal/core/api.rs

```diff
@@ -121,6 +137,12 @@ impl Window {
         self.0.set_rendering_notifier(Box::new(callback))
     }
 
+    /// This function allows registering a callback that's invoked when the user tries to close a window.
+    /// It is emmited when the user tries to "close" a window. The callback has to return a [CloseRequestResponse].
```

Comment:
```suggestion
    /// The callback has to return a [CloseRequestResponse].
```

(the second sentence says the same thing as the first one :-)